pkg/util/pointers.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+// BoolPtr returns a pointer to a boolean value.
```

Comment:
The difficulties of juggling languages ... rephrased, hoping to be less obscure ...